api/integrations/slack/views.py

```diff
@@ -64,7 +64,8 @@ def slack_oauth_callback(self, request, environment_api_key):
         SlackConfiguration.objects.update_or_create(
             project=env.project, defaults={"api_token": bot_token}
         )
-        return Response(status=status.HTTP_204_NO_CONTENT)
+        front_end_redirect_url = request.session.pop("front_end_redirect_url", None)
```

Comment:
should we just make it `request.session.pop("front_end_redirect_url")` since we return 400 if the oauth init endpoint did not provide this in query params

Reply:
I don't know why but it seems better to not crash if `front_end_redirect_url` is not part of the session.